core/drivers/corda-driver/README.md

```diff
@@ -57,11 +57,15 @@ To run the driver, use the following:
 
 The driver gRPC server will be listening on port `9099`.
 
-## Changing the default port configuration
+## Driver configuration
 
 By default, the driver gRPC server listens on port `9099`. To change the port, set 
 the `DRIVER_PORT` as an environment variable. 
 
+Set `DRIVER_RPC_USERNAME` to a rpc user created for driver in the corda network.
```

Comment:
"to a rpc user" --> "to an RPC user". Are there more instructions we can provide here to discover (or create) such a user?